### ISSUE TITLE
Contracts and contracts view page not loading on mobile device

### ISSUE THREAD

**Issue by KaitlinCCSI** [NRGI/resourcecontracts.org#546]
I've now tried to load 3 contracts from OLC and 2 contracts from RC on my mobile device by clicking on their name on the advanced search page. None of them will load at all -- it just takes me to a blank screen with the footer at the bottom.

![img_1619](https://cloud.githubusercontent.com/assets/14825437/10542497/2ecbf1d0-73de-11e5-9a71-0710123a6957.PNG)

**Comment by anjesh** [comment]
Are you still having this problem?

**Comment by KaitlinCCSI** [comment]
No.

However, when I just checked on my phone, the first contract listed on the advanced search page for OLC (http://www.openlandcontracts.org/contract/ocds-591adf-CD0848066778OL/view) only shows a blank white page for the contract with the yellow boxes but no contract content. (See screenshot.) Doesn't seem to be an issue of loading time. This is not a problem when I look at the same contract on my computer, and not a problem for the 2nd and 3rd OLC contract that I checked on my phone.

![img_1707](https://cloud.githubusercontent.com/assets/14825437/11186194/306a9bba-8c46-11e5-986d-a90713d022d7.PNG)
